client/gateway/dappchain_gateway.go

```diff
@@ -43,6 +43,21 @@ type DAppChainGateway struct {
 	Address loom.Address
 }
 
+// AddAuthorisedContractMapping creates a bi-directional mapping between a Mainnet & DAppChain contract without creating a pending mapping
+func (tg *DAppChainGateway) AddAuthorizedContractMapping(from common.Address, to loom.Address, creator *client.Identity, contractTxHash string) error {
```

Comment:
`creator` should be renamed to `gatewayOwner`, and `contractTxHash` can be removed since it's not used.

Reply:
Same has been done.